public/js/add_tables.js

```diff
@@ -0,0 +1,94 @@
+$(document).ready(function() {
+
+  var tableTrack = {
+    addedTables : {}
+  };
+
+  /**
+   *   This function takes the table data and
+   *   updates the DOM, in the #added_tables section.
+   */
+
+  function updateTable() {
+    var keys = Object.keys(tableTrack['addedTables']);
+    var tableView = $("#added_tables");
+    if (keys.length === 0) {
+      tableView.html("");
+      return false;
+    }
+
+    tableView.html("<tr>" +
+      "<th>Number of seats</th>" +
+      "<th>Number of tables</th>" +
+      "</tr>");
+
+    for (var i = 0; i < keys.length; i++) {
+      var row = "<tr><td> " + keys[i] + " </td>";
+      row += "<td> " + tableTrack['addedTables'][keys[i]] + " </td></tr>"
```

Comment:
Mixed double and single quotes.<br>['addedTables'] is better written in dot notation.<br>Missing semicolon.